insights/parsers/vmcore_dmesg.py

```diff
@@ -0,0 +1,43 @@
+"""
+Crash log ``vmcore-dmesg.txt``
+==============================
+
+This module provides ``vmcore-dmesg.txt`` crash logs that are stored in ``/var/crash/[host]-YYYY-MM-DD-HH:MM:SS/``.
```

Comment:
To make the parser can be triggered, it's necessary to add this to the `insights.spec` module: `insights/specs/__init__.py`, `insights/specs/default.py` and `insights/specs/insights_archive.py` and/or `insights/specs/sos_archive.py`

Reply:
If you want to add it to "insights_archive.py" and ensure it can be collected by 'insights-client' tool, the `uploader.json` file in [insights-core-assets](https://github.com/RedHatInsights/insights-core-assets) repo should also be updated correspondingly. 
And a Jira Request should be opened at first as per the Insights Configuration Management Processes, to let the CEE/Engr make a Spec Review.

Reply:
Specs are already defined:
https://github.com/RedHatInsights/insights-core/blob/e17e542dd3010f1dc1293379f351b825c058a6d7/insights/specs/default.py#L817
https://github.com/RedHatInsights/insights-core/blob/5f536d47492e84cd130bd578f177749482e67316/insights/specs/__init__.py#L510
`insights/specs/insights_archive.py` and `insights/specs/sos_archive.py` don't seem to be necessary for this parser